cardstack/templates/page-welcome.php

```diff
@@ -0,0 +1,163 @@
+                    $cardstack_am_video = "";
+                }
+
+                if (CardStackAm::userCanStreamProduct(318)) :
+
+                    if ($cardstack_am_episode == "00") {
+                        echo '<p>Jeśli wolisz napisy bez japońskich tytułów grzecznościowych, przejdź <a href="'
+                            . get_home_url() . '/?altsub=yes">tutaj</a>.</p>';
+                    } else if ($cardstack_am_episode == "00a") {
+                        echo '<p>Napisy bez japońskich tytułów grzecznościowych, z zachodnią kolejnością imion i nazwisk.</p>';
+                    }
+                    ?>
+
+                    <!--data-setup='{"playbackRates": [1, 1.1, 1.2, 2] }'-->
+                    <video id='amagi' class="video-js vjs-16-9 vjs-big-play-centered" style="width: 100%;"
+                           controls="true" oncontextmenu="return false;"
+                           poster="<?php echo $cardstack_am_poster ?>" preload="metadata"
+                           data-setup='{}'>
+                        <source src="<?php echo $cardstack_am_video ?>" type="video/webm" />
+                    </video>
+
+                    <script src="https://static.animagia.pl/video.js"></script>
+
+                    <script>
+                        var vid1 = videojs('amagi');
+                        vid1.on('dblclick', function () {
+                            vid1.requestFullscreen();
+                        });
+                        vid1.on('dblclick', function () {
+                            vid1.exitFullscreen();
+                        });
+                    </script>
+
+                <?php
+                else :
+
+                    CsAmVideo::printFreeFilmPlayer(); ?>
+
+
+                <?php endif; ?>
+
+                <?php comments_template(); ?>
+
+            <?php endwhile; ?>
+        <?php endif; ?>
+
+    </article>
+
+    <article class="page<?php
+    if (get_theme_mod('page_breadcrumbs')) {
+        print(' has-breadcrumbs');
+    }
+    ?>">
+        <div class="container">
+            <?php
+            print("<img src=".get_template_directory_uri()."/templates/Elementario.png alt=\"Amagi\"/>")
+            ?>
+            <div class="image-text">
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris semper dapibus eros.
+                    Quisque et rutrum purus, semper rhoncus purus. Donec libero odio, eleifend a purus
+                    lobortis, accumsan cursus urna. Pellentesque et pharetra metus. Mauris lobortis
+                    metus eu tortor blandit, in viverra est ultricies.
+                </p>
+            </div>
+        </div>
+    </article>
+    <aside style="display: flex; justify-content: space-between; margin: 0 auto 40px auto; max-width: 960px; box-shadow: 0 2px 1px #ccc; background-color: white;">
+        <div style="flex-basis: 33%; text-align: center; padding: 10px 0 0 0;">
```

Comment:
Should be a list, not divs (similar to how the menu is rendered in this theme, so maybe even reuse styles).

Reply:
Done